### DISCUSSION TITLE
How to change root path?

### DISCUSSION THREAD

**Discussion by antoniogfs** [walt-id/.github#89] [Help]
Hi, I'm new to Walt.id and trying to run the wallet api on a local kubernetes cluster.
I would like to run the api on the path like "/wallet/api", but I'm not able to find a way how to change the root path through a configuration value.

Normally on Spring Boot this is possible setting the SPRING_SERVER_SERVLET_CONTEXT_PATH env. variable.
Is there for Kotlin or the application a similar way?

Thank you
Antonio

**Comment by alegomes** [comment]
This forum is no longer being monitored. Please, if you still need our help, submit your question as a [Community Stack issue](https://github.com/walt-id/waltid-identity/issues).

All the best
